src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,39 @@
+package blackjack.domain
+
+import kotlin.math.abs
+import kotlin.math.min
+
+class Player(val name: PlayerName) {
+
+    val cards: Cards = Cards(mutableListOf())
+
+    fun addCard(card: Card?) {
+        if (card != null) {
+            cards.cards.add(card)
+        }
+    }
```

Comment:
일급 컬렉션에서 `getter`를 하지 않아볼까요?

```suggestion
    fun addCard(card: Card?) {
        if (card != null) {
            cards.add(card)
        }
    }
```

Reply:
수정해보았습니다.